ec2/spark_ec2.py

```diff
@@ -1082,11 +1109,11 @@ def real_main():
                     time.sleep(30)  # Yes, it does have to be this long :-(
                     for group in groups:
                         try:
-                            conn.delete_security_group(group.name)
-                            print "Deleted security group " + group.name
+                            conn.delete_security_group(group_id=group.id)
```

Comment:
I would perhaps add a very short comment saying "we must use the group ID here" so that other developers don't change this back to `group.name` and think it works fine because they are testing outside VPCs.